lib/helpers/redis-stubs.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var sinon = require('sinon');
+var redis = require('redis');
+var clientStub = {
+  'get': sinon.stub().callsArg(1),
+  'expire': sinon.stub().callsArg(2),
+  'del': sinon.stub().callsArg(1),
+  'set': sinon.stub().callsArg(2)
+};
+
+var redisCreateClientStub = sinon.stub(redis, 'createClient', function(){
+  return clientStub;
+});
+
+exports.redisCreateClientStub = redisCreateClientStub;
+exports.clientStub = clientStub;
```

Comment:
So long as you don't have a `describe()` inside the file I don't think that `mocha` will run it. i.e. it has nothing to execute so will implicitly ignore it.